### PR TITLE
Feature/vih-5479 dotnet upgrade

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

[VIH-5479](https://tools.hmcts.net/jira/browse/VIH-5479)

### Change description ###

* Update MVC to dotnet core 3.1
* Update npm packages

**Does this PR introduce a breaking change?** (check one with "x")

Install dotnet core sdk 3.1
```
[x] Yes
[] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.Services.Generator/booking-api-csharp.nswag [hunk 1]
```diff
@@ -50,7 +50,7 @@
       "wrapResponseMethods": [],
       "generateResponseClasses": true,
       "responseClass": "BookingsApiResponse",
-      "namespace": "AdminWebsite.BookingsAPI.Client",
+      "namespace": "VideoWeb.Services.Bookings",
```
Comment (rehmanab) [inline AdminWebsite/AdminWebsite.Services.Generator/booking-api-csharp.nswag]: Video Web ?

File: AdminWebsite/AdminWebsite.Services.Generator/booking-api-csharp.nswag [hunk 2]
```diff
@@ -82,9 +82,9 @@
       "typeNameGeneratorType": null,
       "propertyNameGeneratorType": null,
       "enumNameGeneratorType": null,
-      "serviceHost": "http://localhost:5301",
+      "serviceHost": "http://localhost:5300",
       "serviceSchemes": null,
-      "output": "../AdminWebsite.BookingsAPI.Client/BookingsApiClient.cs"
+      "output": "../VideoWeb.Services/BookingsApiClient.cs"
```
Comment (rehmanab) [inline AdminWebsite/AdminWebsite.Services.Generator/booking-api-csharp.nswag]: VideoWeb ?

File: AdminWebsite/AdminWebsite/Startup.cs [hunk 3]
```diff
@@ -103,8 +103,13 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
...
             }
 
+            app.UseRouting();
             app.UseHttpsRedirection();
             app.UseStaticFiles();
+            if (!env.IsDevelopment())
+            {
+                app.UseSpaStaticFiles();
+            }
             app.UseSpaStaticFiles();
```
Comment (rehmanab) [inline AdminWebsite/AdminWebsite/Startup.cs]: Same as Above, doing it twice

Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/Startup.cs]: removed

File: AdminWebsite/AdminWebsite.Services.Generator/booking-api-csharp.nswag (hunk 2 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite.Services.Generator/booking-api-csharp.nswag]: ah was a copy from video-web project when the config got corrupted. ill revert this now

**Pull request merged**